apps/cowswap-frontend/src/modules/hooksStore/containers/PostHookButton/index.tsx

```diff
@@ -21,18 +23,32 @@ export function PostHookButton({ onOpen, onEditHook }: PostHookButtonProps) {
   const { postHooks } = useAtomValue(hooksAtom)
   const removeHook = useRemoveHook()
 
+  const setHooks = useSetAtom(hooksAtom)
+
+  const moveHook = useCallback(
```

Comment:
Code in `PreHookButton` and `PostHookButton` is very similar, I will unify it later